### PR TITLE
Updating Lea's list

### PR DESCRIPTION
**Pull request by pbuttigieg**
Just a suggestion of something you may want to pick up at the store 🐮 
Linked to #5

### CODE REVIEW

File: lists/shoppinglistLea.txt [hunk 1]
```diff
@@ -1,2 +1,2 @@
 birdfood
-fruits
\ No newline at end of file
+fruitswagyu beef
```
Comment (LChWu) [inline lists/shoppinglistLea.txt:2]: insert new line after `fruits`

**Review by LChWu — CHANGES_REQUESTED** [review]
@pbuttigieg thanks, please see the comment

**Comment by pbuttigieg** [comment]
@LChWu many thanks, resolved in https://github.com/pbuttigieg/shoppingList/pull/6/commits/00a1e076fcd70a6e416b3fe09c97573fdb5b0f24

**Review by LChWu — APPROVED** [review]
@pbuttigieg 
many thanks, the line problem is resolved

**Pull request merged**